components/Heroes/heroes.css

```diff
@@ -0,0 +1,79 @@
+.Header {
```

Comment:
then we need to add this standard to contribution page and rename every non capitalize css files, because not all css files have capitalize name in this project